crates/l2/proposer/l1_committer.rs

```diff
@@ -206,6 +219,7 @@ impl Committer {
 
     fn get_deposit_hash(&self, deposit_hashes: Vec<H256>) -> Result<H256, CommitterError> {
         if !deposit_hashes.is_empty() {
+            println!("DEPOSIT HASHES IS NOT EMPTY");
```

Comment:
Should we remove this?

Reply:
yup, removed [d08ff18](https://github.com/lambdaclass/ethrex/pull/2012/commits/d08ff185f97a174c645de97df553408a4ab17a29)